lighthouse-core/test/report/v2/renderer/details-renderer-test.js

```diff
@@ -95,31 +81,6 @@ describe('DetailsRenderer', () => {
       assert.equal(items.children.length, 3, 'did not render children');
     });
 
-    it('renders nested structures', () => {
```

Comment:
why? even if we don't care about blocks we certainly care about text within lists